### PR TITLE
Updates to Android Studio and IntelliJ opening and new Flutter project instructions

### PR DESCRIPTION
**Pull request by anderdobo**
Updates to resolve issues laid out at https://github.com/flutter/website/issues/1928#issuecomment-1555403016

_Description of what this PR is changing or adding, and why:_ This PR corrects the instructions for opening an existing Flutter project, and starting a new Flutter project in Android Studio and IntelliJ.

_Issues fixed by this PR (if any):_ https://github.com/flutter/website/issues/1928

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/tools/android-studio.md [hunk 1]
```diff
@@ -34,10 +34,21 @@ You can create a new project in one of several ways.
 
 ### Creating a new project
 
-To create a new Flutter project from the Flutter starter app template:
+The steps to create a new Flutter project from the Flutter starter app template are 
+slightly different for Android Studio and IntelliJ.
```
Comment (atsansone) [inline src/tools/android-studio.md:38]: **nit:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
Creating a new Flutter project from the Flutter starter app template
differs between Android Studio and IntelliJ.
```

**Review by atsansone — COMMENTED** [review]
@anderdobo ; Would recommend tightening the first sentence, otherwise OK.

**Comment by anderdobo** [comment]
@atsansone updated as suggested. Thanks.

**Review by parlough — APPROVED** [review]
Thanks for these adjustments!

I pushed a few minor formatting and semantic break fixes.

**Pull request merged**